tasks/switch_route.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Patch all apps dest routes with src stack
```

Comment:
Still confusing comment :smile:

Reply:
Now the comment is : `# Patch app dest routes with src stack`